packages/dx-react-grid-bootstrap3/src/plugins/table.jsx

```diff
@@ -41,6 +48,9 @@ Table.NoDataCell = TableNoDataCell;
 Table.NoDataRow = TableRow;
 Table.StubCell = TableStubCell;
 Table.StubHeaderCell = TableStubCell;
+Table.Table = TableComponent;
```

Comment:
Can we use `Table.Root` or `TableContainer` here? @viterobk

Reply:
Table.Root is the another component. Seems that it didn't published yet